clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2412,6 +2454,19 @@ This attribute applies to a loop. Indicates that the loop nest should be
 coalesced into a single loop without affecting functionality. Parameter N is
 optional. If specified, it shall be a positive integer, and indicates how many
 of the nested loop levels should be coalesced.
+
+.. code-block:: c++
+
+  void foo() {
+    int a[10];
+    [[intel::loop_coalesce]] for (int i = 0; i != 10; ++i) a[i] = 0;
+  }
+
+  template<int N>
+  void bar() {
+    [[intel::loop_coalesce(N)]] for(;;) { }
+  }
```

Comment:
Minor: there shall be several nested loops.

Reply:
@MrSidims, Do you prefer to add the nested loop for all template supported attributes? 

Or some thing like this exmaple:
template <int A>
void loop_coalesce() {
  int a[10];
  
  [[intel::loop_coalesce(A)]] for (int i = 0; i != 10; ++i)
      a[i] = 0;
 
  [[intel::loop_coalesce(4)]] for (int i = 0; i != 10; ++i)
      a[i] = 0;
  
  [[intel::loop_coalesce]] for (int i = 0; i != 10; ++i)
      a[i] = 0;
}

Reply:
I mean that loop_coalesce attribute is about loop nesting ("Indicates that the loop nest should be coalesced into a single loop"), so if we want to tell about it in the doc we shall show it.

Reply:
Got it. Thanks for the explanation. I will update the doc.